### ISSUE TITLE
Make it clearer how hive-android works

### ISSUE THREAD

**Issue by fresheneesz** [hivewallet/hive-android#18]
There is some very important missing information in the write ups about hive android. I want to make sure I can backup my coins and that my backup remains valid. These are questions I didn't find easy answers to in the wiki:
- Hive advertises deterministic wallets. Does hive-android use them? I gather the answer is no. This should be made VERY clear. 
- Does hive generate change addresses, or does it use a simple single-address wallet like Mycelium does?
- How can I extract the private key so I can import the address into another wallet program?
- Does hive-android use any kind of network storage?

To be perfectly clear, I think the answers to the above questions should be written in the first place users will go to find information about hive-android. They shouldn't be buried somewhere. And while I want to know the answers, I'm not looking for someone to close this issue after they've informed just me (1 person) about them.

**Comment by javgh** [comment]
Thanks for your feedback. Where would you consider the "first place users will go to find information about hive-android" to be?

Currently the main Hive webpage has a section "Other Hive Products" which in turn links to the Wiki which answers some of the questions above. The main page always has to strike a balance between providing a good overview and answers to crucial questions, while at the same time not overloading the user with information. So I'm not sure how much of these answers could really be integrated into the main page. I agree though, that the wiki could probably be structured better.

We will try to improve on this.

**Comment by fresheneesz** [comment]
I was going to agree that the front-page of Hive is probably not the place for info like this. But looking again, I would say that each platform Hive is on should give the same types of information. For example, on the frontpage, the main hive product (I'm actually not clear what that is) has deterministic wallets. But for Hive android and Hive for Mac, it doesn't say what kind of wallets it uses. I think it should (perhaps with a link to more information if its too much to put on the frontpage).

The front page actually confuses me. I think I understand now after reading the front page more fully (realize that most of your prospective customers will scan your frontpage for what they think they want to know, rather than reading the whole thing). I think the main product is a website. I saw "multiple platforms, now on iOS" - to me this rules out a website. Websites don't get iOS releases. So I really would have to ask you if my assessment is right.

So I'd advise you to treat all your different programs (for different platforms) rather equally, but I think better advise would be to actually _make_ them all equal - ie there should be no difference between android hive, mac hive, and web hive. They should all have the same core features and functionality. This not only makes it easier for your users to understand what hive offers, but allows them to seamlessly switch between your applications cross-device. 

All that said, I am actually curious what the answers to my original questions are.

**Comment by javgh** [comment]
> but I think better advise would be to actually make them all equal - ie there should be no difference between android hive, mac hive, and web hive.

Agreed, that's something we are moving towards, but it's unfortunately not an overnight process.

In the meantime, here are the answers to your questions about Hive Android:
- Hive Android is currently not a deterministic wallet
- Hive Android uses a single Bitcoin address and sends all change back to it. Under the hood it actually supports creating new addresses, but that is currently not exposed in the UI.
- There currently isn't a straightforward way to export your private key for use in other apps. See ticket https://github.com/hivewallet/hive-android/issues/16 for a longer discussion and possible workarounds.
- Hive Android does not store anything on remote servers on its own. When you create a backup, it uses the Android share feature which will usually offer you to save the backup via Google Drive or Dropbox, if you have those services installed and configured.

**Comment by fresheneesz** [comment]
Thanks, thats great information. Can you put the answers to those questions on the hive-android wiki page? Its confusingly blank: https://github.com/hivewallet/hive-android/wiki

I would do it myself, but I can't figure out how to edit those pages.
